packages/node_modules/@ciscospark/react-component-title-bar/src/styles.css

```diff
@@ -7,7 +7,6 @@
 
 .title {
   margin: 0;
-  padding-top: 10px;
```

Comment:
Yea, I made that area flex boxy. When we add presence it should auto align correctly. I took it out because without a presence indicator the name sat above the midline.